luomu-common/src/macaddr.rs

```diff
@@ -16,6 +19,37 @@ impl From<&[u8; 6]> for MacAddr {
     }
 }
 
+impl From<MacAddr> for [u8; 6] {
```

Comment:
For methods returning "raw" bytes of address, it would make sense to state the byte order the address bytes are stored.